lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -172,23 +179,6 @@ class Parser
     private $identVariableExpressions = [];
 
     /**
-     * Checks if a function is internally defined. Used to prevent overwriting
-     * of built-in functions through user-defined functions.
-     *
-     * @param string $functionName
-     *
-     * @return bool
-     */
-    static public function isInternalFunction($functionName)
```

Comment:
This method removal is potentially a BC break, and it needs to be documented in `UPGRADE.md` even though this is internal API